src/test/acceptance/external_connectivity_test.go

```diff
@@ -113,9 +113,6 @@ var _ = Describe("external connectivity", func() {
 			Eventually(cannotProxy, "10s", "1s").Should(Succeed())
 			Consistently(cannotProxy, "2s", "0.5s").Should(Succeed())
 
-			By("checking that the app cannot ping the internet (first time)")
```

Comment:
Is it necessary to remove both of these icmp assertions? Shouldn't these assertions pass on Azure since Azure never allows pings.

If they are deleted, we are losing coverage for making sure that you can't ping even after allowing tcp/udp.

Reply:
@adowns01 Sounds reasonable. Feel free to add back those assertions to the test for the PR to merge.

Reply:
We've updated the test to resolve this issue.